peekingduck/pipeline/nodes/input/live.py

```diff
@@ -47,7 +48,9 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
                 img = resize_image(img,
                                    self.resize_info['width'],
                                    self.resize_info['height'])
-            outputs = {self.outputs[0]: img}
+            outputs = {self.outputs[0]: img, "fps":self.fps_saved_output_video,"filename":"webcamfeed.mp4"}
             return outputs
 
         raise Exception("An issue has been encountered reading the Image")
+
+
```

Comment:
The blank line is failing the linter `sh scripts/linter.sh`

Reply:
i only left 1 newline at the end.